uvicorn/supervisors/watchfilesreload.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, config: Config):
     def __call__(self, path: Path) -> bool:
         for include_pattern in self.includes:
             if path.match(include_pattern):
+                if str(path).endswith(include_pattern):
+                    return True
```

Comment:
Here we check if the path ends with the include pattern, should handle relative and absolute cases.